Modules/Numerics/FEM/include/itkFEMImageMetricLoad.hxx

```diff
@@ -576,9 +576,9 @@ ImageMetricLoad<TMoving, TFixed>::GetPolynomialFitToMetric(VectorType Gpos, Vect
 
   VectorType chebycoefs; // gradient direction
   chebycoefs.set_size(ImageDimension);
-  double chebycoefs0 = 0.0; // the constant term
-  double datatotal = 0.0;
-  double a0norm = 1.0;
+  // NOT USED double chebycoefs0 = 0.0; // the constant term
+  // NOT USED double datatotal = 0.0;
+  // NOT USED double a0norm = 1.0;
```

Comment:
If those variables are really useless, please consider simply removing those lines of code, instead of just commenting them out. 🙏

Reply:
I agree. Restoring the variables is easy with versioned history, if it is ever needed.

Reply:
Removing the comment-out code might actually please the spell checker at the CI, which specifically checks comments! It now complains, saying:

    itkFEMImageMetricLoad.hxx  line: 580  word: datatotal